src/datatypes/map.h

```diff
@@ -136,6 +136,14 @@ int Map_Compare
 	int *disjointOrNull
 );
 
+// merge two maps
+// in case of key collision, the value from 'b' is used
```

Comment:
See my comment on the Map_Merge function documentation in `map.c`
Make sure the documentation in the `c` file is the same as the documentation in the header file.